### DISCUSSION TITLE
Model viewer autoplay goes beyond set camera limits

### DISCUSSION THREAD

**Discussion by henrik-dons** [google/model-viewer#4947] [Q&A]
Hello there,

When I set model viewer camera limits, the autoplay does not respect the set limits. 
It keeps rotating 360 degrees.

Example code:
`<model-viewer touch-action="pan-y" camera-controls="" interaction-prompt="none" src="logo-updarted.glb" alt="3d logo upload" ar="" ar-placement="floor" ar-scale="fixed" min-camera-orbit="-120deg 20deg auto" max-camera-orbit="120deg 140deg auto" auto-rotate="" ar-status="not-presenting" class="">`

Modelviewer version 4.0

Is there way to rotate back when limit is reached, and vice versa?

Let me know!

**Comment by elalish** [comment]
Not currently. It's because technically `autorotate` affects the orientation of the model, not the camera (you can tell if you watch the lighting carefully). I'd recommend you write a `requestAnimationFrame` function to set [`turntableRotation`](https://modelviewer.dev/docs/index.html#entrydocs-stagingandcameras-properties-turntableRotation) as desired instead of using `autorotate`.
